src/app/pages/home/home.component.ts

```diff
@@ -42,4 +44,15 @@ export class HomeComponent implements OnInit {
         break;
     }
   }
+
+  goToDetails(id: string, type: CardTypeEnum): void {
+    switch (type) {
+      case CardTypeEnum.Restaurant:
+        this.router.navigate(['/restaurants', id]);
```

Comment:
please, create and use constants for the routes
```suggestion
        this.router.navigate(['/', RoutingConstants.Restaurants , id]);
```